src/devices/ikea.ts

```diff
@@ -296,6 +296,22 @@ const fzLocal = {
             return {action: `dots_${button}_${action}`};
         },
     } satisfies Fz.Converter,
+    ikea_dots_click_v2_somrig: {
+        cluster: 'tradfriButton',
+        type: ['commandAction1', 'commandAction2', 'commandAction3', 'commandAction4', 'commandAction6'],
+        convert: (model, msg, publish, options, meta) => {
+            const button = utils.getFromLookup(msg.endpoint.ID, {1: '1', 2: '2'});
+            const lookup = {
+                commandAction1: 'initial_press',
+                commandAction2: 'long_press',
+                commandAction3: 'short_release',
+                commandAction4: 'long_release',
+                commandAction6: 'double_press',
+            };
+            const action = utils.getFromLookup(msg.type, lookup);
+            return {action: `dots_${button}_${action}`};
```

Comment:
I think we can remove the `dots_` prefix here (since it's always there). After that this can be merged.

Reply:
Good point!